src/main/resources/db_scripts/event_bookings_add_reserved_by.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE ONLY event_bookings ADD COLUMN reserved_by integer
+        DEFAULT NULL
+        CONSTRAINT event_bookings_users_id_fk
+        REFERENCES users
```

Comment:
Does this not need to be `users.id`, or is Postgres magic?

Reply:
Considering I copypasted this from DataGrip, I would suspect magic, but that's a good point…

Reply:
You learn something new every day: https://stackoverflow.com/a/28560619